### ISSUE TITLE
v0.65 has bugged out hotkeys

### ISSUE THREAD

**Issue by flankytingz** [seerge/g-helper#426]
## NOTE
So I updated to the latest pre-release v0.65 and everything looked fine so I disabled Asus Optimization Service and disabled automatic startup of it. After a laptop restart some of the hot keys were not working.

**Describe the bug**
Keys that do not work:

1. Backlight intensity up
2. Backlight intensity down
3. Aura Switch
4. Track pad toggle
5. Sleep button

**Clear scenario to Reproduce**
Disable Asus Optimization and restart.

**Expected behavior**
The keys should work.

**App Logs**
[log.txt](https://github.com/seerge/g-helper/files/11525017/log.txt)

**Screenshots or screencasts**
I don't have screenshots but for the keyboard light intensity and track pad, the OSD shows up but there are no actual changes. For sleep and aura there is no OSD show up and no effect.

**Desktop (please complete the following information):**
 - OS: Win 11
 - GA402RJ-L8170W (G14 2022)

**Asus software**
- MyAsus installed
- AsusCertService is the only one running

**Comment by seerge** [comment]
@flankytingz if you disabled asus optimization, g-helper has to be running to make hotkeys work. Is it running in background ?

**Comment by flankytingz** [comment]
@seerge I disabled Asus Optimization. G-Helper is always running. Also I just noticed, AniMe Matrix was also no longer working.

I just turned the service back on and then stopped it again, now everything works again.

**Comment by flankytingz** [comment]
@seerge so I tried to reproduce the issue (restarted), I am unable to reproduced it. However, there are still some bugs.

1. The OSD for Touchpad does not alternate, it works but the OSD is the same (I know this doesn't count as a bugged if you only added a single icon).
2. Sleep button still doesn't work. I checked my settings for windows, it is binded to make laptop sleep, I changed it to do nothing and then back to sleep and it still doesn't work.

**Comment by seerge** [comment]
@flankytingz I have figured out what could be a reason. If Asus service is not running, on COLD start (i.e. when you completely turn Off computer and start it again) keyboard backlight won't work at all. It has to be initialized.

I have added initialization in this build - and it should be ok now

Alsop

1. OSD will show touchpad status.
2. What power plan are you using?

[GHelper.zip](https://github.com/seerge/g-helper/files/11525108/GHelper.zip)

**Comment by flankytingz** [comment]
@seerge 

Yes I noticed that off cold, the keyboard bright did not work. It did not auto-dim (so I disabled that option) and the keys wouldn't work until I messed with GHelper.

I am using Silent (on battery). This is assuming that GHelper switched it from Turbo since I was off dock when this happened and my touch pad was disabled so I noticed the issue.

All of the keys work now. I really like the low/mid/high and the on/off and the %brightness changes, so much better.
The sleep key works too.
